### ISSUE TITLE
incorrect or incomplete install instructions

### ISSUE THREAD

**Issue by talltechy** [BellezaEmporium/galaxy-integration-ead#15]
**Describe the bug**
in your readme for this repo, step 3 of the install instructions

> On the folder, use the "install.bat" file to automate the process.
there is no install.bat in the zip file or release page or any mention of it in this repo at all beyond that one spot in the readme

**Comment by Nutzzz** [comment]
Install.bat exists in the (new, better) juno_api and junoapi-test branches. The releases were created from the juno_api branch.

You're right that it either shouldn't be referenced in the (old, "light touch" fork) origin_api branch, or that install.bat should be added to it. And either way, I'd recommend that it should no longer be the default since the releases don't use it, and that fact continues to cause confusion.

EDIT: Also, the README should probably warn that Python and pip must be installed, and be in the PATH.

**Comment by talltechy** [comment]
@Nutzzz 

makes sense, so what is the correct steps to install the plugin now because i see 44 referenced in the new api branch but no actual release zip, do we clone the new api branch and just run the install bat?

**Comment by Nutzzz** [comment]
> i see 44 referenced in the new api branch but no actual release zip, do we clone the new api branch and just run the install bat?

Yes, the latest from the junoapi-test branch has been working pretty well for me. You can either use install.bat or if you don't have Python installed you can replace the files in \<userprofile\>\AppData\Local\GOG.com\Galaxy\plugins\installed\origin_\<ver\> with the ones from this repo's src\ folder.

**Comment by BellezaEmporium** [comment]
That was my mistake. I was actually taking care of the README for all versions, and omitted the fact the Origin API version didn't have an "install.bat" file in.

@Nutzzz We could use GOG Galaxy's python to prevent installing Python system-wide in the process. Though, I'm keeping it into memory.

**Comment by Nutzzz** [comment]
> @Nutzzz We could use GOG Galaxy's python to prevent installing Python system-wide in the process. Though, I'm keeping it into memory.

I thought about that, but IIRC you don't have access to pip using GOG's Python.

**Comment by BellezaEmporium** [comment]
I've added additional disclaimers to prevent any additional confusion.

**Comment by domnsync** [comment]
Hello, I don't have Python so I've replaced all of the files with the new ones manually but it says offline anyway. Is it because EA shut down the origin client? It seems it is not working anymore.

**Comment by BellezaEmporium** [comment]
If you used the "origin_api" version, yes, Origin has been sunsetted in April 17th, 2025. The only available plugins are 0.43 in juno_api, and 0.44 for testing in junoapi-test.

**Comment by domnsync** [comment]
Thanks for the reply. For the 0.43 version, I put the files which are in src folder, into the plugin folder, still shows as offline. What was the time when did you last check if it's working or not?

**Comment by BellezaEmporium** [comment]
> Thanks for the reply. For the 0.43 version, I put the files which are in src folder, into the plugin folder, still shows as offline. What was the time when did you last check if it's working or not?

If it's "Offline" after connecting and restarting Galaxy, it's normal, this release doesn't have any token refreshing method. Most of the new things are in the testing branch. Apparently, some users can't even connect, so i'll add some things to the base branch, see if it goes in properly.

**Comment by domnsync** [comment]
Thanks for the explanation. Actually, there seems to be no way to make it properly work. You're our only hope. I made a star 🌟 for the project and following it. Hope you can manage and work it out. Good luck!

**Comment by BellezaEmporium** [comment]
Please try https://github.com/BellezaEmporium/galaxy-integration-ead/releases/tag/v44

**Comment by domnsync** [comment]
You know what. You're insane. It worked! Game library is showing the exact titles of EA App. Most people are feeling sad for being not able to connect the client, in GOG.com forum site. You saved our day!

Just a little note. It only shows the library titles. It doesn't detect the ones installed on the machine. Or it is not able to start them from the GOG Galaxy launcher.

**Comment by BellezaEmporium** [comment]
> You know what. You're insane. It worked! Game library is showing the exact titles of EA App. Most people are feeling sad for being not able to connect the client, in GOG.com forum site. You saved our day!
> 
> Just a little note. It only shows the library titles. It doesn't detect the ones installed on the machine. Or it is not able to start them from the GOG Galaxy launcher.

Still being a WIP in the test branch. I know why for certain titles, but I still need to implement it properly.

**Comment by domnsync** [comment]
I understand how hard it is, to make it work efficently I mean, so I wish you great luck with your project. By the way, is it okay to share this on GOG.com forums? I'm sure this will be extremely enough for most of the people who are still waiting for a solution.

**Comment by BellezaEmporium** [comment]
I believe there's a post made by Angelblue on GoG that does share this integration.

**Comment by domnsync** [comment]
I just applied the 0.44.2 and it is showing install button which was greyed out before. When you click, it starts the EA App which is awesome. Anyway, it is still not showing in installed tab. Just so you know.

**Comment by BellezaEmporium** [comment]
With the release of 0.44.4, there's an install.bat (or install.command) in windows and mac ZIP files. In the case you had my plugin in Galaxy, an autoupgrade package is also given.

Anyhow, considering the issue's lifespan and the modifications linked to said issue being merged in the repository, I consider the matter to be completed.
